test/descriptor-tests.js

```diff
@@ -0,0 +1,70 @@
+import test from 'tape';
+import compose from '../examples/compose';
+
+test('comopose function pojo (Plain Old JavaScript Object)', nest => {
+  const objDescriptors = [
+    'properties',
+    'deepProperties',
+    'staticProperties',
+    'deepStaticProperties',
+    'propertyDescriptors',
+    'staticPropertyDescriptors',
+    'configuration'
+  ];
+
+  objDescriptors.forEach(descriptorName => {
+    nest.test(`...with pojo descriptor.${ descriptorName }`, assert => {
+      const descriptor = {
+        [ descriptorName ]: {
+          a: 'a'
```

Comment:
This will throw a TypeError for `staticPropertyDescriptors` when `Object.defineProperties` is used in `createStamp`.

> TypeError: Property description must be an object: a